3.3/.sti/bin/assemble

```diff
@@ -8,12 +8,14 @@ set -e
 echo "---> Copying application source ..."
 cp -Rf /tmp/src/. ./
 
-if [ -f setup.py ]; then
-    echo "---> Installing application ..."
-    python setup.py install --user
-elif [ -f requirements.txt ]; then
-    echo "---> Installing dependencies ..."
-    pip install --user -r requirements.txt
+if [[ -f requirements.txt ]]; then
```

Comment:
@soltysh @mfojtik this is a critical part that none of you commented.

Currently we will look for a `setup.py` in the repo, and if it exists we ignore the existence of `requirements.txt`.

In my experience, web apps don't have a `setup.py`, but do have `requirements.txt`. Any reasonable Pythonista uses `requirements.txt` files and not necessarily turn their web project into a distributable/installable package (using `setup.py`)...

So the change I'm proposing here is to consider both files, since [that's what we do in OpenShift v2](https://github.com/openshift/origin-server/blob/0ce2254894ff70577376255e9169fc98d767ea8f/cartridges/openshift-origin-cartridge-python/usr/versions/shared/bin/control#L178-L187).

I would also consider using `python setup.py develop` instead of `python setup.py install`, because that's [what we do in v2](https://github.com/openshift/origin-server/blob/0ce2254894ff70577376255e9169fc98d767ea8f/cartridges/openshift-origin-cartridge-python/usr/versions/shared/bin/control#L186) and `install` just unnecessarily creates more files.

Thoughts?